apps/studio/components/interfaces/Auth/Users/Users.utils.tsx

```diff
@@ -234,7 +232,12 @@ export function getAvatarUrl(user: User): string | undefined {
 
   if (typeof url !== 'string') return undefined
   const isSupported = SUPPORTED_CSP_AVATAR_URLS.some((x) => url.startsWith(x))
-  return isSupported ? url : undefined
+
+  // [Joshen] Only for GH, not entirely sure whats the image transformation equiv for Google
+  const _url = new URL(url)
+  _url.searchParams.set('s', '24')
```

Comment:
This needs a try/catch around it:

<img width="699" alt="Screenshot 2025-02-05 at 15 17 27" src="https://github.com/user-attachments/assets/0447d8df-0ab8-4f60-8cdc-d884a9e65c3a" />